requirements_dev.txt

```diff
@@ -28,3 +28,5 @@ tqdm
 stable_baselines3
 tensorboard
 gym
+jax
+jax[cpu]
```

Comment:
Isn't just `jax[cpu]` enough?

Reply:
it should be, for some reason circle is not picking this up...

Reply:
Ahh, yeah it's because of the circle build cache and tox.  I wish we had a better way for telling `tox` to recreate it's environment but until then you should just bump the "vXX" component of the cache steps in the [circle config](https://github.com/wandb/client/pull/2419/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R51)

@raubitsj any ideas on how to make this happen automatically?  A hacky way to do it would be to store the md5 of `requirements.txt` and `requirements_dev.txt` in a file in the cache, and add the `--recreate` arg to the tox command if that changes...

Reply:
Just had an idea.  We could set [toxworkdir](https://tox.readthedocs.io/en/latest/config.html#conf-toxworkdir) to the hash of `requirements.txt` and `requirements_dev.txt` forcing tox to build a new environment.  This could make the cache get rather large if we keep old tox dirs that aren't used anymore though...

Reply:
Going forward I was going to break up our workers into more granular requirements_dev subsets.
So you would have something for tf2, tf1 pytorch etc